jwst/tweakreg/tests/test_multichip_jwst.py

```diff
@@ -294,7 +295,23 @@ def test_multichip_jwst_alignment(monkeypatch):
 
 
 def test_multichip_alignment_step(monkeypatch):
-    monkeypatch.setattr(tweakreg_step, 'align_wcs', _align_wcs)
+    """
+    This is failing because the XYXYMatch routine is finding bogus x,y shifts.
+    The x,y shift it finds is strongly dependent on the input searchrad value.
+    
+    example log message
+    2024-07-10 12:14:07,340 - stpipe - INFO - Matching sources from 'ext' catalog with sources from the reference 'refcat' catalog.
+    2024-07-10 12:14:07,340 - stpipe - INFO - Computing initial guess for X and Y shifts...
+    2024-07-10 12:14:07,342 - stpipe - INFO - Found initial X and Y shifts of -13.6, 17.2 (arcsec) with significance of 2.861 and 5 matches.
+
+    On the main branch the xyxymatch doesn't seem to do anything at all, i.e.,
+    these lines are not printed at all.
+    Furthermore, hardcoding xyxymatch = None makes this test pass.
+
+    But I can't figure out for the life of me what is different between xyxymatch, or any of the
+    other inputs (or log messages surrounding this one) between this branch and main
+    """
+    monkeypatch.setattr(imalign, 'align_wcs', _align_wcs)
```

Comment:
```suggestion
    monkeypatch.setattr(tweakreg_step.twk, 'align_wcs', _align_wcs)
```

Reply:
Thanks @mcara, sure enough, it works for me locally